docs/fast_data/runtime_management.mdx

```diff
@@ -0,0 +1,147 @@
+      initiate the processing of updates from the Real Time Updater (and the Single View Trigger Generator if active) to generate Single Views. 
+      This sequential approach enhances performance and data consistency.
+
+- **Strategic Ingestion:** achieve better performance and consistency of data by strategically managing the ingestion process, aligning it with the broader data processing pipeline.
+
+### Useful Scenarios
+
+Explore the diverse scenarios where Runtime Management can bring significant benefits to your Fast Data deployment:
+
+- **Optimizing Data Processing Pipelines:** Pause the Single View Creator during high-load data ingestion to ensure optimal system performance. 
+      Resume processing once the ingestion is complete to generate views efficiently.
+- **Sequencing Real-Time and Batch Processing:** Use Runtime Management to sequence the execution of real-time and batch processing components.
+      Pause real-time processing during large batch ingestion, then resume for a balanced data processing approach.
+- **Performance Testing and Simulation:** Simulate different scenarios for performance testing by pausing and resuming microservices.
+      Observe system behavior under various conditions to identify bottlenecks and optimize resource allocation.
+- **Data Consistency Assurance:** Maintain data consistency by pausing the Single View Creator during critical updates to underlying databases.
+      Resume processing after updates to ensure Single Views reflect the most recent and accurate data.
+- **Strategic Resource Allocation:** Dynamically allocate resources by pausing and resuming microservices based on workload and resource availability.
+      Efficiently utilize computing resources and prioritize critical processes during peak demand.
+- **Enhanced System Reliability:** Gracefully handle unexpected situations or system maintenance by pausing specific microservices.
+      Resume operations post-maintenance to minimize downtime and enhance overall system reliability.
+
+## What You Can Do
+
+Explore the capabilities of Runtime Management by executing specific commands tailored to your Fast Data environment.
+
+- ***Pause***. Temporarily halt the consumption of messages on the Single View Creator microservice, be it from *Projection Changes* or *Single View Trigger* messages,
+    so that no aggregation is executed.
+    This command is useful when you want to pause the generation of Single Views, for example, during high-load data ingestion.
+
+- ***Resume***. Restart the consumption of *PC* or *sv-trigger* messages by the Single View Creator, bringing Single Views aggregation process back to active operation.
+
+## Up and Running
+
+To ensure a smooth setup and operation of Runtime Management, make sure you've completed the following tasks:
+
+1. **Set Up the persistence environment:** ensure that you have configured a persistence environment (e.g., MongoDB) for Runtime Management to store its state.
```

Comment:
How can I do it? Is it explained somewhere? If yes can you link it? If no, can you add a paragraph that explains it?